### PR TITLE
blueprints: remove explicit names from initializers.

### PR DESCRIPTION
**Pull request by eventualbuddha**
See https://github.com/emberjs/guides/pull/1654 and https://github.com/ember-cli/ember-cli-legacy-blueprints/pull/54.

### CODE REVIEW

**Comment by rwjblue** [comment]
The blueprint tests are failing (they assert the generated text).

**Comment by eventualbuddha** [comment]
@rwjblue fixed now.

**Comment by Turbo87** [comment]
@rwjblue friendly reminder 😉

**Comment by eventualbuddha** [comment]
@rwjblue ping

**Comment by kellyselden** [comment]
I'm cool with this change.

File: blueprints/initializer/files/__root__/initializers/__name__.js [hunk 1]
```diff
@@ -2,7 +2,4 @@ export function initialize(/* application */) {
   // application.inject('route', 'foo', 'service:foo');
 }
 
-export default {
-  name: '<%= dasherizedModuleName %>',
-  initialize
-};
+export default { initialize };
```
Comment (locks) [inline blueprints/initializer/files/__root__/initializers/__name__.js]: Can we keep this multi-line to minimize diffs (here and for users, especially when adding before/afters)?

Comment (Turbo87) [inline blueprints/initializer/files/__root__/initializers/__name__.js]: actually, the diff will be just as small/large as before unless you start to introduce trailing commas 😉

Comment (locks) [inline blueprints/initializer/files/__root__/initializers/__name__.js]:
```
export default {
  initialize
}
```

-> 

```
export default {
  after: 'ember-data',
  initialize
}};

**Comment by eventualbuddha** [comment]
I don't have time to do this right now, but I believe you all should be able to edit my branch.

**Review by Turbo87 — APPROVED** [review]
this seems like a reasonable change to me

**Comment by stefanpenner** [comment]
> this seems like a reasonable change to me

+1

**Comment by Turbo87** [comment]
@locks @rwjblue @stefanpenner this seems ready to be merged. could someone press the big green button?

**Pull request merged**